### DISCUSSION TITLE
change the order of events in pretixPOS

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3584] [Q&A]
Hello!
Can I change the order the events are displayed in the pretixPOS ?
The manual mentions "Events are sorted in ascending order by beginning".
What if I want to display the events in ascending order by each name ?

**Comment by raphaelm** [comment]
Hi, this is currently not configurable, sorry

**Reply by alemao8** [reply]
Thanks for the update.
Maybe u could add this to future updates. It could be useful to sort things out in pretixPOS when many events occur in the same day..
